src/components/utils/ConfirmationDialog.js

```diff
@@ -17,7 +17,7 @@ import {
     useTheme,
 } from "@material-ui/core";
 
-import { build } from "@cyverse-de/ui-lib";
+import buildID from "components/utils/DebugIDUtil";
```

Comment:
Was this dialog component supposed to be moved to a top-level folder as well?  If so, not sure if we were saving that for another PR.  Looks like a few components are still in the `utils` folder